src/app-layout/visual-refresh-toolbar/interfaces.ts

```diff
@@ -60,4 +60,6 @@ export interface AppLayoutInternals {
   onActiveDrawerResize: (detail: { id: string; size: number }) => void;
   onActiveGlobalDrawersChange: (newDrawerId: string, params: OnChangeParams) => void;
   splitPanelAnimationDisabled?: boolean;
+  expandedDrawerId: string | undefined;
```

Comment:
For widget compatibility, do we want to use `null` as "no drawer expanded" and then `undefined` as "we got an older version with this field not yet defined"